src/refactor_nrepl/artifacts.clj

```diff
@@ -115,12 +115,18 @@
           (sorted-map)
           (get-clojars-artifacts!)))
 
+(defn- prn-str-full
+  [v]
+  (binding [*print-length* nil
+            *print-level*  nil]
+    (prn-str v)))
+
 (defn- update-artifact-cache!
   []
   (let [clojars-artifacts (future (get-artifacts-from-clojars!))
         maven-artifacts (future (get-artifacts-from-mvn-central!))]
     (reset! artifacts (into @clojars-artifacts @maven-artifacts))
-    (spit artifacts-file @artifacts)
+    (spit artifacts-file (prn-str-full @artifacts))
```

Comment:
Why don't you just add the binding here? I think it's going to be much easier to figure out what's going on this way.